notifier/management/commands/forums_digest_flagged.py

```diff
@@ -0,0 +1,75 @@
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from optparse import make_option
```

Comment:
`optparse` is actually part of the Python stdlib, so it should be in its own stanza, before the `django` block of imports.